.vscode/cspell.json

```diff
@@ -31,6 +31,7 @@
   // * Alphabetize the list when making changes so the list is easier for future
   //   users to maintain and reason about.
   "words": [
+    "!respone",
```

Comment:
That's interesting. I didn't know we can override and negate a dictionary word like this.